### PR TITLE
Test which attempts to log in with different combinations of credentials

### PR DESCRIPTION
**Pull request by Zoltekk**
Test which attempts to log in with different combinations of credentials and checks whether it was a successful attempt or not

### CODE REVIEW

File: task1/config.json [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+    "usernames": {
+      "correct_user_1": "standard_user",
+      "correct_user_2": "locked_out_user",
+      "correct_user_3": "problem_user",
+      "correct_user_4": "performance_glitch_user"
+    },
+    "passwords": {
+      "correct_password_1": "secret_sauce"
+    }
+  },
```
Comment (Glovacky) [inline task1/config.json]: Since you're not loading credentials from this file you should remove them.

File: task1/conftest.py [hunk 2]
```diff
@@ -18,4 +19,10 @@ def get_settings():
...
         data = json.load(f)
-    return data["xpaths"]
\ No newline at end of file
+    return data["xpaths"]
+
+@pytest.fixture()
+def get_credentials():
+    with open("config.json") as f:
+        data = json.load(f)
+    return data["credentials"]
```
Comment (Glovacky) [inline task1/conftest.py]: It seems you don't need this fixture anymore

**Review by Glovacky — CHANGES_REQUESTED** [review]

File: task1/config.json (hunk 1 above)
Comment (Zoltekk) [reply task1/config.json]: Removed

File: task1/conftest.py (hunk 2 above)
Comment (Zoltekk) [reply task1/conftest.py]: Removed

**Review by Glovacky — APPROVED** [review]
Good job.

**Pull request merged**